aspnetcore/security/cookie-sharing.md

```diff
@@ -45,6 +45,8 @@ In the `ConfigureServices` method, use the [ConfigureApplicationCookie](/dotnet/
 
 [!code-csharp[](cookie-sharing/sample/CookieAuthWithIdentity.Core/Startup.cs?name=snippet1)]
 
+Data protection keys must be shared among apps. In the sample apps, `GetKeyRingFolderPath` returns the common key storage location. In ASP.NET Core apps, an alternative approach to supplying the key location to [DataProtectionProvider.Create(DirectoryInfo)](/dotnet/api/microsoft.aspnetcore.dataprotection.dataprotectionprovider.create#Microsoft_AspNetCore_DataProtection_DataProtectionProvider_Create_System_IO_DirectoryInfo_) is to configure the data protection service with the [PersistKeysToFileSystem](/dotnet/api/microsoft.aspnetcore.dataprotection.dataprotectionbuilderextensions.persistkeystofilesystem) method. For more information, see [Configuring Data Protection: PersistKeysToFileSystem](xref:security/data-protection/configuration/overview#persistkeystofilesystem).
```

Comment:
Please directly include the sample code:
```
public void ConfigureServices(IServiceCollection services)
{
    services.AddDataProtection()
        .PersistKeysToFileSystem(new DirectoryInfo(@"\\server\share\directory\"));
}
```

Also, let's move this above  ConfigureApplicationCookie and make it the primary recommendation. In core setting DP directly on ConfigureApplicationCookie is the alternative.